archetypes/blog/index.md

```diff
@@ -15,4 +15,5 @@ categories: []
 photo:
   url: https://unsplash.com/photos/wEL2zPX3jDg
   author: Fabio Ballasina
+  alt: "A wide photograph"
```

Comment:
So, if I understand correctly, this is exactly what [`aria-hidden="true"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute) is for; hiding something from a screenreader that doesn't add any actual information. I don't think there are any scenarios in which the wide header image is an info add…

Reply:
We could actually hide the entire `article-header` div, since the photo credit isn't relevant if the photo is hidden (downside, that could be confusing to track down in the future).

Reply:
Sounds good- let's hide the whole div.